.env

```diff
@@ -0,0 +1 @@
+REACT_APP_APIURL="http://127.0.0.1:8000/"
```

Comment:
You probably want an underscore between `API` and `URL`.

Reply:
Sounds good.